app/services/framework_nomis_mappings/alerts.rb

```diff
@@ -15,9 +16,15 @@ def call
   private
 
     def imported_alerts
-      @imported_alerts ||= NomisClient::Alerts.get([prison_number])
+      @imported_alerts ||= begin
+        alerts = NomisClient::Alerts.get([prison_number])
+        nomis_sync_status.set_success
+
+        alerts
+      end
```

Comment:
You could get this on one line if you prefer (I think).. something like this

```suggestion
  @imported_alerts = NomisClient::Alerts.get([prison_number]).tap { nomis_sync_status.set_success }
      end
```

Not sure if that's better or not 🤔